src/core_modules/capture-core/components/WorkingLists/WorkingListsCommon/hooks/useWorkingListsCommonStateManagement.js

```diff
@@ -60,7 +60,7 @@ import type {
 
 const useTemplates = (
     dispatch: ReduxDispatch,
-    { storeId, workingListsType }: { storeId: string, workingListsType: string }) => {
+    { storeId, workingListsType, mainViewConfig }: { storeId: string, workingListsType: string, mainViewConfig?: Object}) => {
```

Comment:
NIT: `mainViewConfig?: MainViewConfig` and `import type { MainViewConfig } from '../../EventWorkingLists';` at the beginning of the file

Reply:
Problem is that since we are in WorkingListsCommon, this file is not only be used by event programs (at least, that is the intention - and I believe it is the case). So, mainViewConfig could vary here depending on the context. From an architectural perspective, WorkingListsCommon shouldn't import from EventWorkingLists because it shouldn't be aware of EventWorkingLists' existence (EventWorkingLists depends on WorkingListsCommon, not the other way around). The proper solution here would be to use Generics. However, since we're missing types here for many other variables for the same reason, and I was concerned about running into "FlowFixMe", I opted out. Would be great to add types, but maybe we do it another ticket (or after migrating to Typescript 😰).